package.json

```diff
@@ -39,7 +40,7 @@
     "vsce:bump": "sh scripts/bump.sh",
     "shell:lint": "shellcheck -x scripts/**/*.sh",
     "test": "sh scripts/e2e.sh",
-    "testScripts": "mocha 'scripts/tests/*.js'",
+    "testScripts": "jest __tests__",
```

Comment:
Even just writing `jest` should suffice, if you rename the test files to `*.test.js`

Reply:
Yes , but the problem then would be, that this would run all tests in the whole repo, not just the script files. You'd then also have to compile first, since the other tests are Typescript files.

Reply:
Jest + TS works, here is a tutorial about that https://basarat.gitbook.io/typescript/intro-1/jest

I agree with the other half about scoping it to only scripts ✅